composer.json

```diff
@@ -19,8 +19,16 @@
         "msgphp/user": "^1.0",
         "symfony/config": "^3.4|^4.0",
         "symfony/dependency-injection": "^3.4|^4.0",
+        "symfony/framework-bundle": "^3.4|^4.0",
```

Comment:
same, can be removed right?

Reply:
It is used [here](https://github.com/msgphp/user-bundle/blob/master/src/DependencyInjection/Extension.php#L59) … but totally up to you.

Reply:
ok. require-dev then?

Reply:
`<insert facepalm emoji here>` … I blame Christmas :rofl: